lineman/app/components/star_toggle/star_toggle.haml

```diff
@@ -1,4 +1,4 @@
-.star-toggle
+.star-toggle{ng-show: 'canStarThread()'}
```

Comment:
We don't support logged out people in the angular app yet. And I think you should be able to star any discussion so long as you're logged in.